apps/api-gateway/src/schema/interfaces/index.ts

```diff
@@ -39,4 +39,22 @@ export interface IOrgAgentInterface {
   walletName: string;
   agentsTypeId: string;
   orgId: string;
-}
\ No newline at end of file
+}
+
+export interface ISchemaInfo {
+  schema: Ischema;
+  schemaId: string;
+  resolutionMetadata: object;
```

Comment:
If possiable we should create interface for `resolutionMetadata`

Reply:
Resolved